backend/python-langchain/main.py

```diff
@@ -49,18 +55,11 @@
 api_url = getenv('LANGCHAIN_UPLOAD_ENDPOINT')
 delete_url = getenv('LANGCHAIN_DELETE_ENDPOINT')
 cors_domains = getenv('CORS_DOMAINS').split(',')
-cookie_domain = getenv('COOKIE_DOMAIN')
 app = Flask(__name__)
 app.secret_key = getenv('SECRET_KEY', '')
 app.config['SESSION_TYPE'] = 'filesystem'
-app.config['SESSION_COOKIE_SAMESITE'] = 'None'
-app.config['SESSION_COOKIE_SECURE'] = True
-app.config['SESSION_COOKIE_DOMAIN'] = cookie_domain
-app.config['REMEMBER_COOKIE_SAMESITE'] = 'None'
-app.config['REMEMBER_COOKIE_SECURE'] = True
-app.config['REMEMBER_COOKIE_DOMAIN'] = cookie_domain
-
-CORS(app, supports_credentials=True, origins=cors_domains)
```

Comment:
@felipegm0911 please return the origins parameter, this is used int he staging environment